<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PullFromDialog</class>
 <widget class="QDialog" name="PullFromDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Synchronize</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="chooseRemoteLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose the remote project you want to synchronize with:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>37</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QWidget" name="paramsWidget" native="true">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>371</width>
     <height>111</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="urlEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="urlLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Path or URL</string>
    </property>
   </widget>
   <widget class="QLabel" name="chooseConflictLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose a conflict resolution mode:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="theirs">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>103</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Conflicting parts of the document will be resolved by favoring the remote version.</string>
    </property>
    <property name="text">
     <string>Theirs</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ours">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Conflicting parts of the document will be resolved by favoring the local version.</string>
    </property>
    <property name="text">
     <string>Ours</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="nameCombo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>251</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>26</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PullFromDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PullFromDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
